<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Config_Verifactu</class>
 <widget class="QDialog" name="Config_Verifactu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>231</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configuración Veri*factu</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Nombre/Razón desarrollador</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="razon_lineEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>NIF:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="nif_lineEdit">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Nombre sistema informático:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="nombre_sistema_lineEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Id sistema informático</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="id_sistema_lineEdit">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Número instalación:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="numero_instalacion_lineEdit"/>
     </item>
     <item>
      <widget class="QCheckBox" name="solo_verifactu_checkBox">
       <property name="text">
        <string>Uso exclusivo  Verifactu</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QCheckBox" name="posible_multi_obligados_checkBox">
       <property name="text">
        <string>Posible uso multi obligados tributarios</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="multiples_obligados_checkBox">
       <property name="text">
        <string>Múltiples obligados tributarios</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Endpoint:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="endpoint_lineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelar_pushButton">
       <property name="text">
        <string>&amp;Cancelar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="aceptar_pushButton">
       <property name="text">
        <string>&amp;Aceptar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelar_pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Config_Verifactu</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>207</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>191</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
